.travis.yml

```diff
@@ -25,7 +25,7 @@ before_install:
     - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
 
     # Get the latest CMake
-    - wget http://www.cmake.org/files/v3.1/${CMAKE_VERSION}.tar.gz
```

Comment:
This has been like this since dc0bc6b4 ... did it stop working? Was it a change by cmake? Just curious.